ComponentKit/Core/Scope/CKComponentScopeFrame.h

```diff
@@ -34,4 +34,16 @@ struct CKComponentScopeFramePair {
 
 @property (nonatomic, strong, readonly) CKComponentScopeHandle *handle;
 
+struct _CKChildScopeFrameRet {
```

Comment:
I think it's fine.

- Move it to the top of the file
- Rename to `CKChildScopeHierarchy` (drop underscore, lose abbreviation, and I like your term "scope hierarchy" below better than "child scope frame")